hw/ip/flash_ctrl/dv/tools/xfile

```diff
@@ -0,0 +1 @@
+SCOPE tb.dut F
```

Comment:
sorry, please move it to dv/tools/xcelium as it's an Xcelium only cfg file. I should say it earlier.

Reply:
Ok,fixed.